packages/renderer/src/lib/kube/KubernetesDashboardResourceCard.svelte

```diff
@@ -0,0 +1,29 @@
+<script lang="ts">
+import type { Component } from 'svelte';
+import { router } from 'tinro';
+
+let {
+  type,
+  Icon,
+  count,
+  link,
+}: {
+  type: string;
+  Icon: Component;
+  count: number;
+  link: string;
+} = $props();
+</script>
+
+<!-- svelte-ignore a11y_missing_attribute -->
+<!-- svelte-ignore a11y_no_static_element_interactions -->
+<!-- svelte-ignore a11y_click_events_have_key_events -->
```

Comment:
issue: do not exclude a11y rules

Reply:
Was copied from Link 😬. I don't remember if there was a reason we didn't use a button at the time, but I've switched this to a button and fixed (removed) these.